sunpy/map/mapbase.py

```diff
@@ -1730,7 +1708,8 @@ def plot(self, annotate=True, axes=None, title=True,
             tr = self._get_lon_lat(self.top_right_coord)
             x_range = list(u.Quantity([bl[0], tr[0]]).to(self.spatial_units[0]).value)
             y_range = list(u.Quantity([bl[1], tr[1]]).to(self.spatial_units[1]).value)
-            imshow_args.update({'extent': x_range + y_range})
+            imshow_args.update({'extent': x_range + y_range,
+                                'aspect': 1})
```

Comment:
I didn't mean to commit this, it doesn't work.

Reply:
Fix it?